server/fqhcs-history/schema/mongoose-schema.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const mongoose = require('mongoose')
+
+const fqhcHistorySchema = mongoose.Schema({
+	fqhcId: mongoose.Schema.Types.ObjectId,
+	field: String,
+	newValue: mongoose.Schema.Types.Mixed,
+	oldValue: mongoose.Schema.Types.Mixed,
+	userId: mongoose.Schema.Types.ObjectId,
+}, {
+	timestamps: true, // createdAt and updatedAt are automatically added
+})
+
+// create model using the schema
+const FQHCHistoryModel = mongoose.model('FQHCHistory', fqhcHistorySchema)
+
+// make available
+module.exports = FQHCHistoryModel
```

Comment:
This is super minor. But since everything in this file is FQHC (without an 's'), do you mind removing it from the directory? So it's fqhc-history, instead of fqhcs-history? That will also match the PRCs.